sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs

```diff
@@ -47,6 +47,17 @@ public static IEnumerable<object[]> ConstructorCreatesDefaultOptionsCases()
             yield return new object[] { new EventProcessorClient(Mock.Of<BlobContainerClient>(), "consumerGroup", "namespace", "hub", credential.Object, default(EventProcessorClientOptions)), "namespace with explicit null options" };
         }
 
+        /// <summary>
```

Comment:
Doesn't look like this is being used at the moment; did you mean to include this?

Reply:
I intended to leave it until the remaining tests marked with #10067 have been debugged and re-enabled. It was the mechanism I used to introduce some concurrency into the test execution which got me a reliable repro for these timing issues. Any concerns?

Reply:
> I intended to leave it until the remaining tests marked with #10067 have been debugged and re-enabled. It was the mechanism I used to introduce some concurrency into the test execution which got me a reliable repro for these timing issues. Any concerns?

None;  I just wasn't sure if something got missed when adding to the index.   Uh, not that I've ever done that before.  :eyes: